src/rust-crypto/backup.ts

```diff
@@ -370,6 +373,51 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
     }
 }
 
+/**
+ * Implementation of {@link BackupDecryptor} for the rust crypto backend.
+ */
+export class RustBackupDecryptor implements BackupDecryptor {
```

Comment:
Naive question: why is this not in `rust-crypto.ts`?

Reply:
It's in the rust backup.ts (mirrors the legacy that has it's implem in legacy backup.ts).
Feels like it belongs there, but no particular technical reason for rust has it just pass it to the wasm olmMachine. The legacy one access some internal classes though.

Reply:
ah, I missed the dir name!